.config/CredScanSuppressions.json

```diff
@@ -122,11 +122,11 @@
       "_justification": "Legitimate UT certificate file with private key"
     },
     {
-      "file": "\\src\\SignalR\\clients\\ts\\FunctionalTests\\testCert.pfx",
+      "file": "\\src\\SignalR\\common\\Shared\\testCert.pfx",
```

Comment:
@BrennanConroy should the renames also happen in release/7.0❔

Reply:
This PR is merging changes from 7.0 to main, so it's already in release/7.0 😕

Reply:
Ooooops, "should the renames also happen in main❔"

Reply:
Yes they should happen in main